### PR TITLE
disable HTTP/2 as a temporary fix for CORE-3441

### PR DESCRIPTION
**Pull request by songgao**
A temporary fix until https://github.com/golang/go/issues/16519 is resolved :)

### CODE REVIEW

File: go/libkb/client.go [hunk 1]
```diff
@@ -135,11 +136,19 @@ func NewClient(e *Env, config *ClientConfig, needCookie bool) *Client {
 		jar, _ = cookiejar.New(nil)
 	}
 
-	var xprt *http.Transport
 	var timeout time.Duration
 
+	xprt := *(http.DefaultTransport.(*http.Transport)) // shallow copying
+
+	// This disables HTTP/2. There's a but introduced between (go1.6, go1.6.3]
```
Comment (maxtaco) [inline go/libkb/client.go]: s/but/bug/

**Comment by maxtaco** [comment]
👍

File: go/libkb/client.go (hunk 1 above)
Comment (songgao) [inline go/libkb/client.go]: Oops! Thanks Max :)

**Comment by songgao** [comment]
@wohlford Jason I think you are hitting https://github.com/keybase/keybase-issues/issues/2114 for that HTTP issue. We have an internal ticket for this but haven't got to it yet. Although for a temporary fix, you could probably revoke [your http proof](https://keybase.io/_/api/1.0/sig/get.json?sig_id=418f6550c4534753c8d6e818966fd42f4f254aa536094d3dd06c2e8cd0bffbe80f) since your https proof for the same domain already shows you own the domain.

**Pull request merged**